components/board/pi/impl/digital_interrupts.go

```diff
@@ -85,16 +85,6 @@ func (i *BasicDigitalInterrupt) Value(ctx context.Context, extra map[string]inte
 	return count, nil
 }
 
-// Ticks is really just for testing.
-func (i *BasicDigitalInterrupt) Ticks(ctx context.Context, num int, now uint64) error {
```

Comment:
:raised_eyebrow: Huh, I didn't realize this was no longer used! Thanks for cleaning it up.